rlog/state-separation.mdx

```diff
@@ -0,0 +1,1255 @@
+
+### a) Components of a Nescience UTXO
+---
+A Nescience UTXO is a critical and versatile component of the private state in the Nescience state-separation architecture. 
+It carries essential information that ensures its proper functionality within private execution, such as the owner, value, private storage slot, non-fungibles, 
+and other cryptographic components. Below is a detailed breakdown of each component and its role in maintaining the integrity, security, and privacy of the system:
+
+* Owner
+The owner component represents the public key of the entity who controls the UTXO. Only the owner can spend this UTXO, ensuring its security and privacy through public key cryptography. 
+This means that the UTXO remains secure as only the rightful owner, using their private key, can generate valid signatures to authorize the transaction. For example, 
+if Alice owns a UTXO linked to her public key, she must sign any transaction to spend it using her private key. This cryptographic protection ensures that only Alice can authorize 
+spending the UTXO and transfer it to someone else, such as Bob.
+
+* Value
+The value in a UTXO represents the balance or asset contained within it. This could be cryptocurrency, tokens, or other digital assets. The value ensures accurate accounting, 
+preventing double-spending and maintaining the overall integrity of the system. For instance, if Alice's UTXO has a value of 10 tokens, this represents her ownership of that amount 
+within the network, and when spent, this value will be deducted from her UTXO and transferred accordingly.
+
+* Private Storage Slot
+The private storage slot is an arbitrary and flexible storage space within the UTXO for Nescience applications. It allows users and smart contracts to store additional private data 
+that is only accessible by the owner. This could be used to hold metadata, smart contract states, or user-specific information. For example, if a smart contract is holding private user data, 
+this information is securely stored in the private storage slot and can only be accessed or modified by the owner, ensuring privacy and security.
+
+* Non-Fungibles
+Non-fungibles within the UTXO represent unique assets, such as NFTs (Non-Fungible Tokens). Each non-fungible asset is assigned a unique serial number or identifier within the UTXO, 
+ensuring its distinctiveness and traceability. For example, if Alice owns a digital artwork represented as an NFT, the non-fungible component of the UTXO will store the unique identifier 
+for this NFT, preventing duplication or forgery of the digital asset.
+
+* Random Commitment Key (RCK)
+The Random Commitment Key (RCK) is a randomly generated number used to create a cryptographic commitment to the contents of the UTXO. This commitment ensures the integrity of the data 
```

Comment:
Replace:
Random Commitment Key

With:
Random commitment key

(and in the above title. Reasoning: there doesn't seem to be a reason this is a proper noun. Also remove (RCK) from the title - we don't need to define the acronym twice.)